docs/features/optin.md

```diff
@@ -0,0 +1,56 @@
+# Opt in to Use Alpha Version Controllers
+
+> `Feature State`: `alpha` as of version v1.125
+
+KCC can be configured to enable alpha version of controllers (referred to as
+"direct" controllers) for reconciling specific resources. This will allow users
+to test the new direct controller code for bug fixes or features not available
+in legacy controllers. It is possible to enable the direct controller code on a
+per-resource basis, by setting an annotation for each custom resource.
+
+
+## Enabling
+
+To enable the direct controller for a specific resource, update the resource and
+specify the annotation:
+
+```yaml
+metadata:
+  annotations:
+    alpha.cnrm.cloud.google.com/reconciler: "direct"
```

Comment:
One question users may have is that if this annotation accept other values, like "terraform" or "dcl". Could you add some explanation to that?

Reply:
ook. updated